testing-basics-garden-project/garden-test001.js

```diff
@@ -0,0 +1,4 @@
+/* We will be writing tests against the code for the Garden program found in the starter-files directory.
+   Let's write tests against the Garden requirements first.
+   To start, import the Garden object from the src directory
+*/

```

Comment:
This reads really well. Short, sweet, and to the point.

It's probably review at this point, but on the off chance that someone skips previous projects, we can include a quick example of importing an object or module. Also made a couple of minor formatting tweaks:

```
/* 
  We will be writing tests against the code for the `Garden` program found in the starter-files directory.
  Let's write tests against the `Garden` requirements first.
  To start, import the `Garden` object from the `src` directory.
  For example, here's how you would import `Plane` from the same directory:
  `import { Plane } from "../src/Plane";`
*/
```